ResearchKit/Common/ORKChoiceAnswerFormatHelper.m

```diff
@@ -117,7 +121,7 @@ - (id)answerForSelectedIndexes:(NSArray *)indexes {
             [array addObject:value];
         }
     }
-    return [array copy];
+    return array.count > 0 ? [array copy] : ORKNullAnswerValue();
```

Comment:
I came across this line when dealing with `Save` and `Clear answer` button states. `ORKImageSelectionView` and `ORKTextChoiceCellGroup` use this helper method to set their `answer` property when a touch event comes in. Once the event got handled, we compare the new answer value to the original answer value to check if the answer did actually change (which is one requirement for enabling the save button). 

When the user skipped this step initially the original answer is `ORKNullAnswerValue()` (`NSNull*)